opensearch/src/main/java/org/opensearch/sql/opensearch/setting/OpenSearchSettings.java

```diff
@@ -90,7 +91,7 @@ public class OpenSearchSettings extends Settings {
   public static final Setting<?> QUERY_SIZE_LIMIT_SETTING =
       Setting.intSetting(
           Key.QUERY_SIZE_LIMIT.getKeyValue(),
-          LegacyOpenDistroSettings.QUERY_SIZE_LIMIT_SETTING,
+          IndexSettings.MAX_RESULT_WINDOW_SETTING,
```

Comment:
I forgot what does this `fallbackSetting` argument mean. What's the difference between this and changing the default value 200 in `LegacyOpenDistroSettings`?

Reply:
In don't think we should touch the default value of any LegacyOpenDistroSettings, although the effects are same.